source/app/service-providers/cli-provider.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @ignore
+ * BEGIN HEADER
+ *
+ * Contains:        cli-provider class
+ * CVM-Role:        Service Provider
+ * Authorr:         Felix Nüsse
+ * License:         GNU GPL v3
+ *
+ * Description:     This class handles the cli-arguments.
+ *                  It can be used to query arguments, as
+ *                  long as they are defined in the options object.
+ *
+ * END HEADER
+ */
+
+import { app } from 'electron'
+import path from 'path'
+
+export const DATA_DIR: string = 'data-dir'
```

Comment:
Since these are constant primitives, I think we can remove the type annotations, since these will be picked up easily by Typescript